src/components/pages/areas-manage/AreasManage.js

```diff
@@ -135,7 +135,7 @@ class AreasManage extends React.Component {
                     />
                 </div>
               </div>
-              <Button className="c-button">Save</Button>
+              <Button className="c-button" disabled={this.props.saving}>Save</Button>
```

Comment:
It should currently disable to button until there is text inside the field through Form validations. 

For the edit area, agreed. Will add.